.github/workflows/ci-testing.yml

```diff
@@ -0,0 +1,31 @@
+name: Continuous testing & Linting
+
+on: [push]
+
+jobs:
+  build:
+
+    runs-on: ubuntu-latest
```

Comment:
I prefer fedora for running in different python versions. Over time they have the more stable setup than ubuntu. But this is personal taste :-)

Reply:
I don't have an opinion here. Since Ubuntu works so far in Travis and GitHub actions for other projects I think it's good to leave for now. If any issues come up we could switch.